.github/workflows/typescript_publish.yml

```diff
@@ -38,6 +38,8 @@ jobs:
         run: pnpm run build
 
       - name: Publish
-        run: pnpm publish --provenance --access public
+        run: |
+          pnpm run prepublish
+          pnpm publish --recursive --no-git-checks --provenance --access public
```

Comment:
`--recursive` tells pnpm to publish all workspace packages (which in this case, is just the package at `src/` which is the `@coinbase/cdp-sdk` package.

`--no-git-checks` is [used by changesets](https://github.com/changesets/changesets/blob/main/packages/cli/src/commands/publish/npm-utils.ts#L79-L80) (which we will be leaning more heavily on once they have commit & tag signing), and it is needed here because we will be transiently modifying the `src/package.json` as a result of running `pnpm run prepublish` above.